helm_deploy/cla-frontend/templates/ingress.yaml

```diff
@@ -8,10 +8,16 @@ metadata:
   name: {{ $fullName }}
   labels:
     {{- include "cla-frontend.labels" . | nindent 4 }}
-  {{- if .Values.ingress.whitelist }}
   annotations:
     nginx.ingress.kubernetes.io/whitelist-source-range: "{{ include "cla-frontend.whitelist" . }}"
-  {{- end }}
+    nginx.ingress.kubernetes.io/server-snippets: |
+      location /socket.io {
+        proxy_pass http://nodejs';
```

Comment:
I am not sure whether we need to carry this across because I don't think we define nodejs upstream config as we did previously https://github.com/ministryofjustice/cla_frontend/blob/master/docker/nginx.conf#L110

Reply:
Ah ok cool I just blindly copied it without understanding :D